### DISCUSSION TITLE
自定义顶栏的操作逻辑两点建议

### DISCUSSION THREAD

**Discussion by 233YUZI** [the1812/Bilibili-Evolved#4238] [功能建议 / Ideas]
![image](https://github.com/the1812/Bilibili-Evolved/assets/69708373/75db7fb7-d1bf-4d8e-8465-78a985796dcb)
能否把这种数字点改成点击后才会消失呢？现在的hover消失很难受啊，动态弹出数字的频率太高，我一般结合反广告插件屏蔽这种数字的，现在的hover逻辑导致我无法在插件中选中这个数字，一碰到就消失
---------------------
还有消息下拉栏的逻辑也有问题，来消息后如果我从主页直接点“消息“进入消息中心页面后，数字点不会消失，必须要我点到消息来源的子项才行，比如“回复我的”“系统通知”
![image](https://github.com/the1812/Bilibili-Evolved/assets/69708373/a519ef9a-cd67-4d86-a95d-e6d5009a4d65)
---------------------
而且消息中心页面的顶栏又变成了这样，子项的数字点还在
![image](https://github.com/the1812/Bilibili-Evolved/assets/69708373/b7396846-6c6c-45cf-873c-4fce72f7d9bd)

**Comment by Geeyun-JY3** [comment]
> 还有消息下拉栏的逻辑也有问题，来消息后如果我从主页直接点“消息“进入消息中心页面后，数字点不会消失，必须要我点到消息来源的子项才行，比如“回复我的”“系统通知”

这有啥问题么，没点开对应子项的消息查看就得留着顶部的数字点啊，原版顶栏不就这样么。

> 而且消息中心页面的顶栏又变成了这样，子项的数字点还在

你的意思是在[消息中心](https://message.bilibili.com/)页面时，自定义顶栏的消息顶部的数字点不应在子项还有数字点时就消失？这倒确实，原版顶栏确实不会这样。
